index.md

```diff
@@ -11,6 +11,8 @@ In addition, the Julia developer community is contributing a number of [external
 Julia programs are organized around [multiple dispatch](http://docs.julialang.org/en/release-0.3/manual/methods/#man-methods); by defining functions and overloading them for different combinations of argument types, which can also be user-defined.
 For a more in-depth discussion of the rationale and advantages of Julia over other systems, see the following highlights or read the [introduction](http://docs.julialang.org/en/release-0.3/manual/introduction/) in the [online manual](http://docs.julialang.org).
 
+{% include juliacon-player.md %}
```

Comment:
Sure - I thought about that, but figured someone else can probably make a better decision than me on what to write (and where to place this).

As a matter of fact, I think I'll add a commit that removes this addition, and then this PR can be merged without changing the look of the site. Then, someone else can write a full section of text and use this mixin to do the actual inclusion of the video.